client/menu/menu.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
+    <link rel="stylesheet" href="../../third_party/codemirror-5.19.0/lib/material.css">
+    <link rel="stylesheet" href="menu.css">
+    <script src="../../third_party/angular-1.6.1/angular.min.js"></script>
+  </head>
+  <body>
+    <div class="tie-menu-question-list-wrapper">
```

Comment:
Alright. I kept one of the cards since I feel like we might still be able to use the structure that I put in place for the future cards, and then added a greeting text. It's fairly simple/not stylized (I would add a logo but I don't think we have a PNG of the file) so let me know if you think I should add a bit more CSS to style it up.